From 895bfd9da2b85bf24c5cfffc41da1cbbea5f31de Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Sat, 14 Jul 2012 12:52:09 +0900
Subject: [PATCH] Add 'tasks' section (Closes: #670429).

This also arranged the list as one name per line, in plain text, alphabetically
sorted, for better tracking.
---
 policy.sgml |   71 ++++++++++++++++++++++++++++++++++++++++++++++-------------
 1 file changed, 56 insertions(+), 15 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 52dbb26..a62f01f 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -714,21 +714,62 @@
 	<p>
 	  The Debian archive maintainers provide the authoritative
 	  list of sections.  At present, they are:
-	  <em>admin</em>, <em>cli-mono</em>, <em>comm</em>, <em>database</em>,
-	  <em>devel</em>, <em>debug</em>, <em>doc</em>, <em>editors</em>,
-	  <em>education</em>, <em>electronics</em>, <em>embedded</em>,
-	  <em>fonts</em>, <em>games</em>, <em>gnome</em>, <em>graphics</em>,
-	  <em>gnu-r</em>, <em>gnustep</em>, <em>hamradio</em>, <em>haskell</em>,
-	  <em>httpd</em>, <em>interpreters</em>, <em>introspection</em>,
-	  <em>java</em>, <em>kde</em>, <em>kernel</em>, <em>libs</em>,
-	  <em>libdevel</em>, <em>lisp</em>, <em>localization</em>,
-	  <em>mail</em>, <em>math</em>, <em>metapackages</em>, <em>misc</em>,
-	  <em>net</em>, <em>news</em>, <em>ocaml</em>, <em>oldlibs</em>,
-	  <em>otherosfs</em>, <em>perl</em>, <em>php</em>, <em>python</em>,
-	  <em>ruby</em>, <em>science</em>, <em>shells</em>, <em>sound</em>,
-	  <em>tex</em>, <em>text</em>, <em>utils</em>, <em>vcs</em>,
-	  <em>video</em>, <em>web</em>, <em>x11</em>, <em>xfce</em>,
-	  <em>zope</em>.  The additional section <em>debian-installer</em>
+admin,
+cli-mono,
+comm,
+database,
+debug,
+devel,
+doc,
+editors,
+education,
+electronics,
+embedded,
+fonts,
+games,
+gnome,
+gnu-r,
+gnustep,
+graphics,
+hamradio,
+haskell,
+httpd,
+interpreters,
+introspection,
+java,
+kde,
+kernel,
+libdevel,
+libs,
+lisp,
+localization,
+mail,
+math,
+metapackages,
+misc,
+net,
+news,
+ocaml,
+oldlibs,
+otherosfs,
+perl,
+php,
+python,
+ruby,
+science,
+shells,
+sound,
+tasks,
+tex,
+text,
+utils,
+vcs,
+video,
+web,
+x11,
+xfce,
+zope.
+	  The additional section <em>debian-installer</em>
 	  contains special packages used by the installer and is not used
 	  for normal Debian packages.
 	</p>
-- 
1.7.10

